### PR TITLE
KOGITO-2592: Include persistence and events profiles in Jenkinsfile

### PR DESCRIPTION
**Pull request by Sgitario**
JIRA: https://issues.redhat.com/browse/KOGITO-2592
Description: Include persistence and events profiles in the Jenkinsfile.

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

File: Jenkinsfile.deploy [hunk 1]
```diff
@@ -68,8 +68,23 @@ pipeline {
...
+                }
+            }
+        }
+        stage('Build kogito-examples with events') {
+            steps {
+                // Use a separate dir for events to not overwrite the test results
+                checkout([$class: 'GitSCM', branches: [[name: env.BRANCH_NAME]], browser: [$class: 'GithubWeb', repoUrl: 'git@github.com:kiegroup/kogito-examples.git'], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'kogito-examples-events']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'kie-ci-user-key', url: 'git@github.com:kiegroup/kogito-examples.git']]])
+                dir("kogito-examples-events") {
+                    script {
+                        maven.runMavenWithSubmarineSettings('clean install -Pevents', false)
```
Comment (radtriste) [inline Jenkinsfile.deploy:87]: can you add the ` saveReports()` as a post ?

File: Jenkinsfile.deploy.new [hunk 2]
```diff
@@ -85,8 +85,23 @@ pipeline {
...
+            }
+        }
+        stage('Build kogito-examples with events') {
+            steps {
+                // Use a separate dir for events to not overwrite the test results
+                checkout([$class: 'GitSCM', branches: [[name: env.BRANCH_NAME]], browser: [$class: 'GithubWeb', repoUrl: 'git@github.com:kiegroup/kogito-examples.git'], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'kogito-examples-events']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'kie-ci-user-key', url: 'git@github.com:kiegroup/kogito-examples.git']]])
+                dir("kogito-examples-events") {
+                    script {
+                        maven.runMavenWithSubmarineSettings('clean install -Pevents', false)
                     }
```
Comment (radtriste) [inline Jenkinsfile.deploy.new:105]: can you add the saveReports() as a post ?

**Comment by radtriste** [comment]
What about `Jenkinsfile` as you did on apps and examples ?

File: Jenkinsfile.deploy.new (hunk 2 above)
Comment (Sgitario) [reply Jenkinsfile.deploy.new:105]: Done

File: Jenkinsfile.deploy (hunk 1 above)
Comment (Sgitario) [reply Jenkinsfile.deploy:87]: Done

**Comment by Sgitario** [comment]
> What about `Jenkinsfile` as you did on apps and examples ?

I missed this file. Updated.

**Pull request merged**